README.md

```diff
@@ -175,6 +177,39 @@ Then in the console, check the coordinates when you move the marker (should only
 
 ---
 
+### [leaflet-wms]
+
+Much the same as leaflet-map above, but for wms services and uses `source` for the source service, 
+`layer` for the layer name and `crs` for the coordination system (only supported by 
+leaflet: EPSG:3857 and EPSG:4326)
+
+All three values are mandatory; though defaulting to a mundialis service.
+```
+[leaflet-wms source="https://your/wms/service?" layer="yourLayer" crs="EPSG:3857" zoom=1]
+```
+
+#### [leaflet-wms] Options:
+
+| Option            | Default                                   |
+| ----------------- | ----------------------------------------- |
+| `source`          | http://ows.mundialis.de/services/service? |
```

Comment:
Http?

Reply:
Its from the Leaflet WMS-Tutorial. You prefer https? Works, too.

https://leafletjs.com/examples/wms/wms.html